insomniac/params.py

```diff
@@ -48,6 +48,9 @@ def parse_arguments(all_args_dict, starter_conf_file_path):
 
         refresh_args_by_conf_file(args)
 
+    if args.speed is not None:
```

Comment:
I'd move this param check into `insomniac/session.py` file.
This file should hold very 'dummy' logic which is to parse the params, and forward them to the params-owners.
On this scenario, the param owner is 'session.py' because he is the one that telling 'I have a param called `speed`'.

So under `insomniac/session.py`, there is a function called `set_session_args`, and the check should take a part in there.